fuzzers/041-clk-hrow-pips/clk_table.py

```diff
@@ -0,0 +1,30 @@
+HCLKS = 24
```

Comment:
Can we wrap this in a function? Is there a reason this needs to be done on import? If so, please add a comment.

Reply:
I assume you are referring to CLK_TABLE, and not `HCLKS = 24`?